frontera/contrib/messagebus/zeromq/socket_config.py

```diff
@@ -1,29 +1,62 @@
 # -*- coding: utf-8 -*-
+"""
+Contains the SocketConfig class
+"""
+from socket import getaddrinfo, gaierror
 
 
 class SocketConfig(object):
-
-    hostname = None
-    base_port = None
-
-    def __init__(self, hostname, base_port):
-        self.hostname = hostname
-        self.base_port = base_port
+    """
+    Converts address to IPv4 or IPv6 or * and returns the necessary socket
+    addresses.
+    NOTE: When using * it defaults to IPv4
```

Comment:
I only mentioned it in the code because IPv6 will probably not work properly (neither the test is implemented yet, nor was I able to try it out on my own machine).
But I will move it to the broker command line help and somewhere in the docs (I do not think it belongs to the `ZMQ_ADDRESS` setting since this setting is not supposed to be used with `*` anyway).
Do you want me to add a note saying that frontera has only been tested with IPv4?